cmd/kubeapps-apis/plugins/helm/packages/v1alpha1/globals_vars_test.go

```diff
@@ -485,6 +502,15 @@ var (
 		}),
 	}
 
+	addRepoReqWithInterval = &corev1.AddPackageRepositoryRequest{
+		Name:            "bar",
+		Context:         &corev1.Context{Namespace: "foo", Cluster: KubeappsCluster},
+		Type:            "helm",
+		Url:             "http://example.com",
+		NamespaceScoped: true,
+		Interval:        "99m",
```

Comment:
Again, not sure that's valid. We could even simplify things by just using a drop-down selection in the UX (and validating valid values of 1, 2, ..., 15, 20, 30 etc.